components/ScrollToTop/index.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+import { ArrowUp } from '@/components/Icons';
+
+export default function ScrollToTop() {
+  const [isVisible, setIsVisible] = useState(false);
+
+  const handleScroll = () => {
+    const { scrollY } = window;
+
+    scrollY > 300 ? setIsVisible(true) : setIsVisible(false);
```

Comment:
```suggestion
    setIsVisible(scrollY > 300);
```
이것보다 한눈에 보기 좋게 하기 위해서 풀어쓰신건가용?

Reply:
> 이것보다 한눈에 보기 좋게 하기 위해서 풀어쓰신건가용?

저는 사고의 흐름대로 흘러가는 게 가독성 측면에서 좋다고 생각해서 위처럼 풀어서 쓰는 편이기는 한데요,
지수님이 제안해 주신 방법도 라인수가 짧아지니까 오히려 가독성이 좋다고 생각이 드네요 ㅎㅎ